cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -506,6 +506,20 @@ class fixed_width_column_wrapper : public detail::column_wrapper {
 template <typename Rep>
 class fixed_point_column_wrapper : public detail::column_wrapper {
  public:
+  /**
+   * @brief Construct a non-nullable column of the decimal elements in the range `[begin,end)`.
+   *
+   * Example:
+   * @code{.cpp}
+   * // Creates a non-nullable column of INT32 elements with 5 elements: {0, 2, 4, 6, 8}
+   * auto elements = make_counting_transform_iterator(0, [](auto i) { return i * 2;});
+   * auto w = fixed_width_column_wrapper<int32_t>(elements, elements + 5, scale_type{0});
+   * @endcode
+   *
+   * @param begin The beginning of the sequence of elements
+   * @param end   The end of the sequence of elements
+   * @param scale The scale of the elements in the column
```

Comment:
template parameter doc is missing.
Edit: I see this missing in other existing docs, so I'm not sure if it's required. Pretty sure it's in the reference docs format.